### ISSUE TITLE
[Container Request]: mummer

### ISSUE THREAD

**Issue by kapsakcj** [StaPH-B/docker-builds#310]
### Contact Details

curtis.kapsak@theiagen.com

### What tool are you looking for and why?

mummer for calculating ANI values

Looks like mummer v4.x.x is a separate code repo from mummer 3.x.x and older. Probably best to go with latest version, I think?
https://github.com/mummer4/mummer

It's on [bioconda](https://bioconda.github.io/recipes/mummer4/README.html?highlight=mummer#package-package%20&#x27;mummer4&#x27;) and [biocontainers exist for mummer4 on quay.io](https://quay.io/repository/biocontainers/mummer4?tab=tags), but would be nice to have a StaPH-B docker image available too.

**Comment by kapsakcj** [comment]
#327  PR should close this Issue, but I'd like to keep it open for now. I'll create a second dockerfile that includes the RGDv3 fasta files to be used for ANI. I think this will be very useful for those in the community that are working with enteric pathogens

**Comment by kapsakcj** [comment]
started dev branch `cjk-mummer-RGD` for this. Needs testing & polishing before starting a PR

Additionally started PR #399 to add RGDv2 to the fastANI docker image for similar functionality

**Comment by kapsakcj** [comment]
closed via #405
